src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/data/UserDetail.kt

```diff
@@ -16,14 +17,23 @@ data class UserDetail(
   @Schema(description = "Last name of the user", example = "Smith", required = true) val lastName: String,
   @Schema(description = "Active Caseload of the user", example = "BXI", required = false) val activeCaseloadId: String?,
   @Schema(description = "Status of the user", example = "OPEN", required = false) val accountStatus: AccountStatus?,
-  @Schema(description = "Type of user account", example = "GENERAL", required = true) val accountType: UsageType = UsageType.GENERAL,
-  @Schema(description = "Email addresses of user", example = "test@test.com", required = false) val primaryEmail: String?,
+  @Schema(
+    description = "Type of user account",
+    example = "GENERAL",
+    required = true
+  ) val accountType: UsageType = UsageType.GENERAL,
+  @Schema(
+    description = "Email addresses of user",
+    example = "test@test.com",
+    required = false
+  ) val primaryEmail: String?,
   @Schema(description = "List of associated DPS Role Codes", required = false) val dpsRoleCodes: List<String>,
   @Schema(description = "Account is not locked", required = false) val accountNonLocked: Boolean?,
   @Schema(description = "Credentials are not expired flag", required = false) val credentialsNonExpired: Boolean?,
   @Schema(description = "User is enabled flag", required = true) val enabled: Boolean,
   @Schema(description = "User is admin flag", required = false) val admin: Boolean?,
   @Schema(description = "User is active flag", required = true) val active: Boolean,
+  @Schema(description = "Staff Status", example = "ACTIVE", required = true) val staffStatus: StaffStatus?,
```

Comment:
If required = true should this not be nullable?

Reply:
thanks Karen..changed